### PR TITLE
updates made to entities

### PR DESCRIPTION
**Pull request by yourgirlcodes**
Updates made to  Groups, Relationships, Customers- activities and entity
activity indicators files moved in order to keep with the folder structure 'style'

### CODE REVIEW

**Comment by iBaryo** [comment]
@yourgirlcodes as we got more indicators now, i put them in a common directory - i see that you moved them out, why?

**Comment by yourgirlcodes** [comment]
Initially, it looked better to have Indicator separated from ActivityIndicator - however, logically that is incorrect. 

see my update - I just changed the folder name to have an uppercase 'I' to match the folder layout. There is then an 'ActivityIndicator' directory too, within.
I added 'Purpose' directory to keep to the folder structure

File: entities/Purpose/Purpose.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import {Id, StaticEntity} from './common';
-import {SegmentName} from "./Segment";
-import {ActivityIndicatorName} from "./indicators/ActivityIndicator";
-import {ActivityFieldName, ProfileFieldName} from "./common/Field";
-import {ActivitySchemaName} from "./Schema";
+import {Id, StaticEntity} from '../common';
```
Comment (iBaryo) [inline entities/Purpose/Purpose.ts]: rename this file to `index.ts` so imports could continue to be from `/Purpose`

File: entities/index.ts [hunk 2]
```diff
@@ -2,13 +2,13 @@ export * from './Workspace';
...
-export * from './Purpose';
+export * from './Purpose/Purpose';
 export * from './Connector';
 export * from './Event';
 export * from './Action';
 export * from './Segment';
-export * from './indicators/ActivityIndicator';
-export * from './indicators/CalculatedIndicator';
+export * from './Indicator/ActivityIndicator';
+export * from './Indicator/CalculatedIndicator';
```
Comment (iBaryo) [inline entities/index.ts]: rename directory back to `indicators` - it's grouping of several actual config-entities

File: entities/CustomerEntity/index.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import {Id, WithDetails, WithId} from "../common";
+import {SchemaType, WithSchema} from "../Schema";
+import {WithProtectedFields} from "..";
+
+export type relationId = Id;
+
+export interface CustomerEntity extends SchemasBase, WithProtectedFields {}
+
+export interface SchemasBase extends WithId, WithSchema, WithDetails {
```
Comment (iBaryo) [inline entities/CustomerEntity/index.ts]: typo, `SchemaBase`?

Comment (yourgirlcodes) [reply entities/CustomerEntity/index.ts]: typo - will update. thanks for noticing

File: entities/index.ts (hunk 2 above)
Comment (yourgirlcodes) [reply entities/index.ts:11]: so when you group config-entities or anything not model/model-like then you keep to lowercase? We reserve uppercase directories for grouping of Model/Class entities?

File: entities/Purpose/Purpose.ts (hunk 1 above)
Comment (yourgirlcodes) [reply entities/Purpose/Purpose.ts]: will do, thank you

Comment (yourgirlcodes) [reply entities/Purpose/Purpose.ts]: will do, thank you

**Pull request merged**